File: long contest/NAIPC/nwerc2017practice/practice/cubebits/submissions/accepted/rgl.java
import java.util.*;

public class rgl {
  public static void main(String[] args) {
    final Scanner sc = new Scanner(System.in);
    for (int i = sc.nextInt(); i --> 0;) {
      final long x = sc.nextLong();
      System.out.println(Long.toString(x, 3));
    }
  }
}

File: long contest/NAIPC/nwerc2017practice/practice/cubebits/submissions/accepted/rgl.kt
import java.io.*
import java.util.*

fun main(args: Array<String>) {
  val br = BufferedReader(InputStreamReader(System.`in`))
  for (i in 0 until br.readLine().toInt()) {
    System.`out`.println(br.readLine().toLong().toString(3))
  }
}
